src/components/Dashboard/AddressListSm.vue

```diff
@@ -167,8 +167,8 @@ export default class AddressListSm extends Mixins(EntityFilterMixin, CommonMixin
   @Prop({ default: false })
   private coaPending: boolean
 
-  // EntityTypes Enum
-  readonly EntityTypes: {} = EntityTypes
+  // Enum definition for use in template.
+  readonly EntityTypes = EntityTypes
```

Comment:
This looks much better, i think we started doing this elsewhere as well, since you mentioned this wasn't required in the latest version of TS.